presto-docs/src/main/sphinx/admin/feature-toggles.rst

```diff
@@ -9,8 +9,27 @@ Though, if you think the behavior of a feature is unexpected, don't hesitate to
 
-+--------------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|Name of property          | Default value | Description                                                                                                                                                |
-+==========================+===============+=======================================================================================================+++==================================================+
-|optimizer.reorder-windows | true          |Allow reordering windows in order to put those with the same partitioning next to each other. This may sometimes allow minimizing number of repartitionings.|
-+--------------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
++---------------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|Name of property           | Default value | Description                                                                                                                                                |
++===========================+===============+=======================================================================================================+++==================================================+
+|optimizer.reorder-windows  | true          |Allow reordering windows in order to put those with the same partitioning next to each other. This may sometimes allow minimizing number of repartitionings.|
++---------------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|experimental.spill-enabled | true          |Try spilling memory to disks to avoid exceeding memory limits for the query. See more detailed description below.                                           |
++---------------------------+---------------+------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. _spilling-toggle:
+
+Spilling to disks
+-----------------
+
+Spilling works by offloading memory to disks. This can allow some queries with large memory footprint
+to pass at the cost of slower execution times. Currently spilling is supported only for aggregations
+so this property will not reduce memory usage required for joins, window functions and sorting.
+
+Beware that this is experimental feature and should be used with care.
+
+Current state of the implementation has a side effect that will cause all queries with aggregations to
```

Comment:
Suggest:
"Currently, all queries with aggregations will slow down after enabling spilling. It is recommended to use the spill session property to selectively turn on spilling only for queries that would run out of memory otherwise."
Also, add a link to the session property in the text above